app/views/layouts/_admin_sidebar.html.haml

```diff
@@ -107,7 +107,7 @@
         - if can? :update, @conference.sponsorship_levels.build
```

Comment:
This builds a sponsorship_level hence the `.persisted?` check.A better alternative is to check `@conference.sponsorship_levels.count > 0` instead of `!@conference.sponsorship_levels.empty?  && @conference.sponsorship_levels.first.persisted?`. Should I change it count?